src/python-common/ceph/deployment/translate.py

```diff
@@ -0,0 +1,70 @@
+import logging
+from ceph.deployment.drive_group import DriveGroupSpec
+from ceph.deployment.drive_selection.selector import DriveSelection
+
+logger = logging.getLogger(__name__)
+
+
+class ToCephVolume(object):
+
+    def __init__(self,
+                 spec,  # type: DriveGroupSpec
+                 selection  # type: DriveSelection
+                 ):
+
+        self.spec = spec
+        self.selection = selection
+
+    def run(self):
+        """ Generate ceph-volume commands based on the DriveGroup filters """
+        try:
+            data_devices = [x.path for x in self.selection.data_devices()]
+        except AttributeError:
+            data_devices = [x for x in self.selection.data_devices()]
+        db_devices = [x.path for x in self.selection.db_devices()]
+        wal_devices = [x.path for x in self.selection.wal_devices()]
+        journal_devices = [x.path for x in self.selection.journal_devices()]
+
+        if not data_devices:
+            return None
+
+        if self.spec.objectstore == 'filestore':
+            cmd = "lvm batch --no-auto"
+
+            cmd += " {}".format(" ".join(data_devices))
```

Comment:
Feeling slightly uneasy that we enforce `data_devices` to never contain spaces. 

If `cmd` would be a list, this would actually look a bit simpler:

```suggestion
            cmd += data_devices
```

Reply:
I don't think that `data_devices` would ever have spaces since it will be parsed from a drivegroup. I'll look into transforming that to a list though.